Composer/packages/client/src/components/Tree/index.tsx

```diff
@@ -3,17 +3,16 @@
 
 /** @jsx jsx */
 import { jsx } from '@emotion/core';
-import { PropTypes } from 'prop-types';
 
 import { container } from './styles';
 
-export const Tree = (props) => (
+type ITreeProps = {
+  variant: string;
+  children: Element;
+};
+
+export const Tree = (props: ITreeProps) => (
```

Comment:
In order to preserve component names in the bundled code, export components like:

```ts
const ComponentName: React.FC<ComponentPropsInterface> = (props) => { ... }
export { ComponentName };
```